src/providers/LocalVideoProvider/index.tsx

```diff
@@ -39,17 +38,21 @@ const LocalVideoProvider: React.FC = ({ children }) => {
   }, [audioVideo]);
 
   const toggleVideo = useCallback(async (): Promise<void> => {
-    if (isVideoEnabled || !meetingManager.selectedVideoInputDevice) {
+    if (isVideoEnabled || !meetingManager.selectedVideoInputTransformDevice) {
```

Comment:
If someone does not want to use the background blur and the VideoInputControl with bg blur, then would this still work as expected?